doc/source/train/config_guide.rst

```diff
@@ -69,8 +69,8 @@ are :ref:`not tunable <air-tuner-search-space>`.
     :end-before: __sync_config_end__
 
 
-Checkpoint configuration (``CheckpointConfig``)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Checkpoint configurations in Train (``CheckpointConfig``)
```

Comment:
do we need to specify "in Train" for everything? It's already under the Train docs, and makes the headings more verbose.

Reply:
@amogkam yeah, I understand it looks verbose, but this is to boost doc search results.

Currently, if you search "checkpointing ray train" you just get nothing meaningful back.